docs/articles_en/documentation/openvino-ir-format/operation-sets/operations-specifications/matrix/Inverse_14.rst

```diff
@@ -0,0 +1,135 @@
+.. {#openvino_docs_ops_matrix_Inverse_14}
+
+Inverse
+=======
+
+
+.. meta::
+  :description: Learn about Inverse-14 - a matrix operation that computes the inverse or adjoint of one or multiple input matrices.
+
+**Versioned name**: *Inverse-14*
+
+**Category**: *Matrix*
+
+**Short description**: *Inverse* operation computes either the inverse or adjoint (conjugate transposes) of one or more square invertible matrices.
+
+**Detailed description**: *Inverse* operation computes the inverse of a square matrix. The operation uses LU decomposition with partial pivoting to compute the inverses.
```

Comment:
Can we describe here a reference algorithm for LU decomposition? Or at least leave a link to it?

Reply:
For now I have added the link to the algorithm description, I will add the full algorithm pseudo code after CF!